source/partials/release_notes/_release-17-5-0.md.erb

```diff
@@ -1,3 +1,7 @@
+<h4>Important</h4>
+
+There has been a critical security vulnerabilty identified with this release, fixes for the same have been provided in 17.7.0. If you are on 17.5.0, immediately upgrade to 17.7.0 to keep your GoCD sever and agent secure.
```

Comment:
Small tweak: There is a critical security vulnerability identified in this release. A fix has been provided in version 17.7.0. If you are on 17.6.0, please upgrade immediately to 17.7.0 to keep your GoCD sever and agent secure.

Also, can we link to 17.7.0?